lib/resources/virtualization.rb

```diff
@@ -25,11 +24,14 @@ class Virtualization < Inspec.resource(1)
     "
 
     def initialize
+      @virtualization_data = Hashie::Mash.new
+
       unless inspec.os.linux?
-        skip_resource 'The `virtualization` resource is not supported on your OS yet.'
-      else
-        collect_data_linux
+        raise Inspec::Exceptions::ResourceSkipped,
```

Comment:
instead of this can we just put `supports: 'linux'` at the top?

Reply:
Looks like `supports: unix` is on master. Will remove these bits and rebase.